manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml

```
@@ -1,18 +1,16 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
-PackageIdentifier: Rws.TradosStudio
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Trados.TradosStudio2021
 PackageVersion: 16.1.4276
 PackageLocale: en-US
-Publisher: Rws
+Publisher: SDL Limited (a part of the RWS Holdings Plc group)
 PublisherUrl: https://www.rws.com/
 PublisherSupportUrl: https://www.rws.com/support/
-#PrivacyUrl: 
-#Author: 
 PackageName: Trados Studio 2021
```

```suggestion
PackageName: SDL Trados Studio 2021 SR1
```
This must match Control Panel for `winget list` mapping.

---

Matched to ARP